torch/_inductor/codegen/wrapper.py

```diff
@@ -1013,6 +1013,15 @@ def define_kernel(
         self.header.splice(f"\n\n{metadata_comment}{name} = {kernel}")
 
     def define_user_defined_triton_kernel(self, kernel, configs, kwargs):
+        import triton
+
+        # Horrible Hack
+        # Triton emits triton.language.float32 as triton.language.fp32
+        # so our code generation fails as fp32 is not a real thing
```

Comment:
Submit a PR to triton to add an fp32 alias (or fix the repr print)?  Then add a TODO in this code linking to the PR (so we can remove the hack in the future).

Reply:
Created https://github.com/openai/triton/pull/3342